.travis.yml

```diff
@@ -5,8 +5,8 @@ branches:
 script:
   - bundle exec rake
 rvm:
-  - 2.0
   - 2.1
   - 2.2
   - 2.3
   - 2.4
+  
```

Comment:
Oh, that's not good. That's [what I wrote about](https://github.com/panthomakos/timezone/pull/79#discussion_r152751873).

Reply:
This was caused by the merge conflict resolver in GitHub. I removed the trailing line on `master`.